### ISSUE TITLE
Is it possible to Add Custom fields to other meta boxes?

### ISSUE THREAD

**Issue by samjco** [elliotcondon/acf#754]
It would be nice to make a location rule for this I think.

**Comment by elliotcondon** [comment]
Hi @samjco 

Thanks for the feature request.
Could you please flesh this out a bit with some more description and screenshots to show a real world example?

**Comment by samjco** [comment]
Hey @elliotcondon 

No problem. So if you have a look at this post:

https://wordpress.stackexchange.com/questions/111825/add-custom-field-to-existing-meta-box

You will see that the user wanted to add default meta fields to the author metabox. 

So I am wishing to add fields to any metabox that has been init with Wordpress.

**Comment by elliotcondon** [comment]
Thanks for the reply.

The best way to achieve this (at the moment) is to use jQuery and move the elements around on page load. 

Here is a "some-what" related tutorial for moving elements: https://www.advancedcustomfields.com/resources/moving-wp-elements-content-editor-within-acf-fields/

I'll have a think about adding this into core, but am also conscious that WP are removing metaboxes completely in the new Gutenberg editor.

**Comment by samjco** [comment]
So far
 `get_meta_boxes( );` And `print_r($wp_meta_boxes`)

Shows a list of metaboxes registered. 

So I was thinking that this could be a Location Rule for ACF

**Comment by samjco** [comment]
Yeah that’s right about Gutenberg. So moving using us would work. 

But it will still be of some use to those that using the Classic Editor.
 Because developing in dev mode and not having https, Gutenberg acts crazy when updating.
